<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formSlidePlotSettings</class>
 <widget class="QDialog" name="formSlidePlotSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>410</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Plot Settings</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>350</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Minimum Wavelength</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinBoxMinWave">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>26</y>
     <width>141</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>15000.000000000000000</double>
   </property>
   <property name="value">
    <double>0.000000000000000</double>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinBoxMaxWave">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>66</y>
     <width>141</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>15000.000000000000000</double>
   </property>
   <property name="value">
    <double>1100.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Maximum Wavelength</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinBoxXFrame">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>106</y>
     <width>141</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>15000.000000000000000</double>
   </property>
   <property name="value">
    <double>50.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>X-Frame</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinBoxYFrame">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>146</y>
     <width>141</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>15000.000000000000000</double>
   </property>
   <property name="value">
    <double>50.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y-Frame</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>194</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>X-Step</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinBoxXStep">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>190</y>
     <width>141</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>15000.000000000000000</double>
   </property>
   <property name="value">
    <double>10.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>181</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Y-Step</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="spinBoxYStep">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>236</y>
     <width>141</width>
     <height>29</height>
    </rect>
   </property>
   <property name="minimum">
    <double>0.000000000000000</double>
   </property>
   <property name="maximum">
    <double>15000.000000000000000</double>
   </property>
   <property name="value">
    <double>50.000000000000000</double>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>formSlidePlotSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>formSlidePlotSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
